integration-tests/test-calico/test_calico.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eux
+
+# This job expect awscli and juju to be configured
+
+CHARM_CHANNEL=${CHARM_CHANNEL:-"candidate"}
+
+function cleanup {
+  juju destroy-controller aws-test-vpc --destroy-all-models -y || true
+  ./cleanup-vpc.sh
+}
+trap cleanup EXIT
```

Comment:
One thing I'd be worried about is the possibility of leaving orphaned VPCs behind.

If we cancel the job, does the cleanup still run?

If the process dies due to e.g. hardware failure, there's certainly no chance for the cleanup to run then. Do we have a fallback? For example, in the case of Juju models we have this fallback job: https://ci.kubernetes.juju.solutions/view/Maintenance/job/clean%20up%20leftover%20instances/

Reply:
I would be satisfied with creating tags ([see comment](https://github.com/juju-solutions/kubernetes-jenkins/pull/209/files#r199189598)) and also calling `./cleanup-vpc.sh` in a post-build step of the jenkins job.

Reply:
Added the tags and the post build jobs.